test/asyncio_tests/test_aiohttp_gridfs.py

```diff
@@ -222,7 +222,6 @@ async def test_content_type(self):
             ('bar', 'octet-stream'),
             ('bar.png', 'png'),
             ('ht.html', 'html'),
-            ('jscr.js', 'javascript'),
```

Comment:
I don't think we loose anything by removing javascript here. We already test various other types.